Rakefile

```diff
@@ -93,31 +93,39 @@ namespace :acceptance do
   require_relative './spec/support/acceptance/helpers'
   include TargetHelpers
 
-  desc 'Provisions the VMs. This is currently just the master'
```

Comment:
Yeah, we shouldn't be removing VM tasks. Any extra docker tasks should be additive.